turbinia/e2e/teardown-gce.sh

```diff
@@ -0,0 +1,45 @@
+#/bin/bash
+# This script will cleanup and teardown the Turbinia Terraform setup after
+# the Turbinia e2e test is finished.
+echo "Tear down Terraform Turbinia infrastructure."
+
+
+if [ $# -ne  2 ]
+then
+  echo "Not enough arguments supplied, please provide project and zone."
```

Comment:
very minor nit, `please provide the project and zone name`

Reply:
done.